ci-operator/step-registry/cucushift/installer/check/aws/outpost/cucushift-installer-check-aws-outpost-ref.yaml

```diff
@@ -0,0 +1,20 @@
+ref:
+  as: cucushift-installer-check-aws-outpost
+  from_image:
+   namespace: ocp
+   name: "4.14"
+   tag: upi-installer
+  commands: cucushift-installer-check-aws-outpost-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: EXPECTED_OUTPOST_NODE
+    default: "3"
+  - name: EXPECTED_MTU_OVN
```

Comment:
So this expected value is equal to `CLUSTER_NETWORK_MTU`, right?

Reply:
If right, can well directly use `CLUSTER_NETWORK_MTU` for checking?

Reply:
I used the same logic as `cucushift-installer-check-aws-localzone`, sometimes `CLUSTER_NETWORK_MTU` is not set, e.g. MTU degradation, so it's better that user gives a expect value explicitly

Reply:
If that, can we also set `EXPECTED_MTU_OVN` in provision chain together with `CLUSTER_NETWORK_MTU`, so that we can quickly get know what env should be set in provision chain yaml file.

Reply:
Done.